### PR TITLE
Feature/highschool cesb

### LINKED ISSUE #422: add high school graduates for CESB
add option under "Income the same" - screen M2c in figma
result mapping updated in Calvin's spreadsheet under May 8 tab https://docs.google.com/spreadsheets/d/11DSoj5_NDiznVITuSDcedxfNSMSGTwTm9B0J1Jwv1Ww/edit#gid=663536631 

check that high school graduates are included in CESB benefit card in the first bullet - Previously missing from French translation

i believe we are still holding this in staging

### PR DESCRIPTION
**Pull request by katedee**
This PR:
- adds back CESB (but not CSSG)
- adds a CESB option for highschool students to `unchanged-income` page (M2c)

Resolves #422 

| French  | English |
| ------------- | ------------- |
| image | image |
| image |  image |

### CODE REVIEW

**Comment by katewilhelm** [comment]
Need to change benefit results order. CESB should be second after CERB. I've started showing order in Figma to the left of the first results screen.

Because I used struck-out text in figma, there's an extra "or high school" in the first bullet that needs to be deleted. This only affects English. I'll stop doing that and just use magenta to indicate revised text since it seems you're copying the whole bullet or paragraph.

Also, I noticed it's doing that bug again where if I'm on the results page in English, toggle to French, go back to English, then click start over, it brings me to the start page in French. Not sure if it's just a staging thing or a live thing, but I believe this bug arose before and was fixed. I'll also post this in the dev channel too.

**Comment by katewilhelm** [comment]
Looks great! Approved! I'll create another issue for the bug

File: views/benefits/cesb-en.njk [hunk 1]
```diff
@@ -6,33 +6,51 @@
...
+              <strong>OR $2,000 per month if you have disabilities</strong>
+            </li>
+            <li>
+              <strong>OR $2,000 per month if you have dependants</strong>
+            </li>
+          </ul>
+        </p>
+      </div>
+    </li>
   </ul>
```
Comment (dsamojlenko) [inline views/benefits/cesb-en.njk:27]: oh, I should have done the OAS card like this

**Comment by CalvinRodo** [comment]
@dsamojlenko Can you make that change as a quick PR to this branch? So we minimize downtime...

**Pull request merged**